src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -669,10 +669,8 @@ internal static void HighlightPort(Step stepInfo, StepUIAutomation uiAutomationD
 
             //Inside the NodeView try to find the ItemsControl that contains Input ports or Output ports
             var itemsControlPort = GuideUtilities.FindChild(byOriginNode, portHighlighted) as ItemsControl;
-
             if (itemsControlPort == null) return;
-
-            var inPorts = itemsControlPort.Items.Cast<PortViewModel>().ToList();
+            var inPorts = itemsControlPort?.Items.Cast<PortViewModel>().ToList();
```

Comment:
It is already checking for null in line 672.

Reply:
I can remove the check